src/runtime_src/core/tools/xbutil2/CMakeLists.txt

```diff
@@ -82,7 +82,7 @@ target_link_libraries(${XBUTIL2_NAME}
   )
 
 if (NOT WIN32)
-  target_link_libraries(${XBUTIL2_NAME} PRIVATE pthread uuid dl)
+  target_link_libraries(${XBUTIL2_NAME} PRIVATE pthread boost_thread uuid dl)
```

Comment:
We don't want this dependency unless there is absolutely now other way.

Reply:
Removed the thread. Now no thread functionality is implemented.